app/controllers/api/avatars.php

```diff
@@ -11,12 +11,15 @@
 use Utopia\Cache\Adapter\Filesystem;
```

Comment:
Please remove all comments from the code, we can always use other branches or tags to revert.